ledger/src/sigverify_shreds.rs

```diff
@@ -377,6 +375,7 @@ pub fn sign_shreds_gpu_pinned_keypair(keypair: &Keypair, cache: &RecyclerCache)
     vec
 }
 
+#[cfg(test)]
```

Comment:
nit: This function doesn't need to be `pub` either. Since you changed the other two, I think it makes sense to change this one too

Reply:
Good point, done